### ISSUE TITLE
[Shaman] Grounding Totem having a mind of it's own

### ISSUE THREAD

**Issue by Dorkazera** [SunwellWoW/Sunwell-TBC-Bugtracker#3913]
**Description:** 
Grounding Totem decides to take a day off randomly.

**How it works:** 
Grounding Totem just some times (might be range related issue, have to do some more testing) decides to not work, the debuff goes on and off what seems randomly.
See the video, this happens alot.

**How it should work:** 
Grounding Totem should be consumed, no matter what the range is between the totem and the spell caster, as long as the player targeted by the spell is under the aura of said Grounding Totem.

**Source (you should point out proofs of your report, please give us some source):** 

**Short Video:**
https://www.youtube.com/watch?v=NyujgQUWFVc

**Comment by username-is-not-taken** [comment]
Oh don't worry, bug reports have been submitted regarding grounding totem since three weeks ago (excluding pre-launch - if you wan't to count pre-launch it's since the middle of October), but it's not a critical problem so who cares =)

And I doubt it's a range issue by the way; see below:

![grounding](https://user-images.githubusercontent.com/45629295/50550195-75dd9280-0c6b-11e9-8ed2-35974a8e4d9a.png)


![fb1](https://user-images.githubusercontent.com/45629295/50550196-7aa24680-0c6b-11e9-9d24-a736cf2505cd.png)


![fb2](https://user-images.githubusercontent.com/45629295/50550198-7f66fa80-0c6b-11e9-9dd9-d0fe2d95d9da.png)


![fb3](https://user-images.githubusercontent.com/45629295/50550199-8857cc00-0c6b-11e9-8a4a-c4371cffba78.png)

**Comment by username-is-not-taken** [comment]
Can we get an update on the progress of fixing this? PvP is basically unplayable when it's completely random if grounding will take a spell or not.

**Comment by username-is-not-taken** [comment]
@Dootz if you need more combat logs/ examples. Both are from today:

![grounding1](https://user-images.githubusercontent.com/45629295/51502630-340dbb00-1dd7-11e9-95ce-4d80e35129aa.png)

![grounding2](https://user-images.githubusercontent.com/45629295/51502635-3839d880-1dd7-11e9-85c3-873fed8cacaa.png)



![grounding3](https://user-images.githubusercontent.com/45629295/51502642-3b34c900-1dd7-11e9-874f-33fb802e942b.png)

![grounding4](https://user-images.githubusercontent.com/45629295/51502644-3d972300-1dd7-11e9-84a5-cf085d6bc8bc.png)

**Comment by username-is-not-taken** [comment]
![combatlog_grounding](https://user-images.githubusercontent.com/45629295/53288603-73d71200-378a-11e9-92c3-e3dac4082cd1.png)

I couldn't fit the entire combat log on the screen since it concerns 35s of combat and cba to take multiple screens and put them together, but what's seen above is an arena game vs disc/slsl where the grounding totem was down for 35(!!!) seconds without taking a single spell (it probably would've been down for the entire 45s duration if the arena would've lasted that long).

@akrom23 @Dootz 
Can we get a critical label on this issue? It hasn't worked for almost three months now and its crucial both for pvp and pve. For instance, in my last Vashj kill (17th february) I had four shock blasts go through grounding totem.

**Comment by akrom23** [comment]
@username-is-not-taken Tell me if the buff from the totem was on you al the time?

**Comment by username-is-not-taken** [comment]
I wasn't paying attention to that I'm afraid, but I very much doubt it was since most of the times when it's not consumed by spells is when it doesn't show up in the buff bar. And I'm very confident that I was within range, and line of sight, of the totem for a large majority of those 35 seconds (if not 100% of the time)

**Comment by username-is-not-taken** [comment]
![grounding123](https://user-images.githubusercontent.com/45629295/53293396-685c0900-37d3-11e9-9ae2-471b8231cbfb.jpg)

Played a BG about an hour ago and here's one example. As you can see I don't have the grounding buff and I have a moonfire dot which has been on me for 7 seconds, when my grounding has been down for 10 seconds i.e. it should've hit the grounding.

Here's the combat log (didn't think of expanding it though, will do next time):

![grounding456](https://user-images.githubusercontent.com/45629295/53293428-d56f9e80-37d3-11e9-8cc6-7eb9efc7ec45.png)

**Comment by akrom23** [comment]
@username-is-not-taken Tell me one more thing - enemy target must be in LoS with the totem in order to destroy it?
Eg.
You are in LoS with totem, got a buff from totem. 
You are in LoS with the enemy target - casting a spell on you.
Totem is **NOT** in LoS with the enemy target - what should happen?
The same question is about when enemy target is out of range of totem.

**Comment by username-is-not-taken** [comment]
I tried to google a bit regarding this (I'm quite new to the class, so I'm not completely certain myself) and I could only find two helpful things. This post: https://eu.battle.net/forums/en/wow/topic/3523322456

"Grounding can be placed anywhere, infact I recommend trying to place it out of sight"
Sure, it's not a comment from the TBC days, but the changelog found on wowwiki (https://wowwiki.fandom.com/wiki/Grounding_Totem) doesn't suggest there has been any changes to how it works with LoS.

And this video:
https://www.youtube.com/watch?v=OzXDQtpdslE

I think 1:45 in the video answers your question about the range. The grounding is destroyed by earth shock as the lightning bolt is traveling to it, and the grounding itself is definitely over 25y range away from the caster, which is the earth shock cast range.
The video also mentions that grounding had the same mechanics through all expansions, which makes the first link more legit.

To summarize, it seems like the only thing that matters is if the player targeted by a harmful spell is within range of the grounding totem, which should be 30 yards according to this link: https://www.wowhead.com/spell=8178/grounding-totem-effect

"Effect Apply Area Aura: Magnet
Radius: 30 yards"
